<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtInputWindow</class>
 <widget class="QWidget" name="QtInputWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>421</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>529</width>
    <height>541</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtInputWindow</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY나무M</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>새 노래 추가</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>추가</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancleButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>341</width>
     <height>290</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="topMargin">
     <number>7</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>고유번호</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="input_id">
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>제목</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="input_title">
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>가수</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="input_singer">
      <property name="inputMask">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>작곡가</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="input_composer">
      <property name="inputMask">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>장르</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Unknown</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dance</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ballad</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Idol</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pop</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Indie</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rap/Hiphop</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rock</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>R&amp;B</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Classic</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>가사</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTextEdit" name="textEdit">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Gulim';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ChangeButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>변경</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lyricfileButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <zorder>ChangeButton</zorder>
  <zorder>label</zorder>
  <zorder>AddButton</zorder>
  <zorder>cancleButton</zorder>
  <zorder>formLayoutWidget</zorder>
  <zorder>lyricfileButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>AddButton</sender>
   <signal>clicked()</signal>
   <receiver>QtInputWindow</receiver>
   <slot>newMusicCreate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancleButton</sender>
   <signal>clicked()</signal>
   <receiver>QtInputWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ChangeButton</sender>
   <signal>clicked()</signal>
   <receiver>QtInputWindow</receiver>
   <slot>ChangeMusicInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lyricfileButton</sender>
   <signal>clicked()</signal>
   <receiver>QtInputWindow</receiver>
   <slot>lyricfileOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newMusicCreate()</slot>
  <slot>ChangeMusicInfo()</slot>
  <slot>lyricfileOpen()</slot>
 </slots>
</ui>
